### PR TITLE
Script/generate metadata

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Generate runtime docs for v38 (Mainnet) and one for v39 (Porcini)

Closes https://centralitydev.atlassian.net/browse/TRN-247

### CODE REVIEW

**Comment by aidan-starke** [comment]
We have made a new file structure, can you generate these in this pattern?
```
 - docs
    - node-interfaces
        - runtime-38
        - runtime-39
```

**Comment by KarishmaBothara** [comment]
> We have made a new file structure, can you generate these in this pattern?
> 
> ```
>  - docs
>     - node-interfaces
>         - runtime-38
>         - runtime-39
> ```

Updated

File: yarn.lock
Comment (ken-centrality) [inline yarn.lock:1]: we are using `pnpm` so don't need the `yarn.lock`

**Review by ken-centrality — APPROVED** [review]
LGTM!

**Pull request merged**